thcrap/src/bp_file.h

```diff
@@ -30,6 +30,9 @@ typedef struct {
 	size_t pre_json_size;
 
 	void *game_buffer;
+
+	// Hack
+	bool disable;
```

Comment:
add a more meaningful comment. this flag is needed to prevent reentrancy/recursion when calling file_load in anm_get_size, is that correct?

Reply:
Yes